packages/cli/project.json

```
@@ -20,7 +20,11 @@
       "dependsOn": ["build"]
     },
     "exec": {
-      "command": "node dist/packages/cli/index.js",
+      "command": "node dist/packages/cli/index.js --configPath=./packages/cli/code-pushup.config.ts",
+      "dependsOn": ["build"]
+    },
+    "npx": {
+      "command": "npx dist/packages/cli",
```

Perhaps we could merge these?

```suggestion
      "command": "npx dist/packages/cli --configPath=./packages/cli/code-pushup.config.ts",
```

I figured out how to get rid of "ENOENT: no such file or directory, chmod '/.../.npm/_npx/.../node_modules/@quality-metrics/cli/src/bin.js'" errors, `npx clear-npx-cache` worked for me. So I think we can safely use `npx` instead of `node` to execute packages now :crossed_fingers:

---

done